components/public-api-server/pkg/webhooks/stripe_test.go

```diff
@@ -27,6 +27,7 @@ const (
 	invoiceFinalizedEventType   = "invoice.finalized"
 	customerCreatedEventType    = "customer.created"
 	customerSubscriptionDeleted = "customer.subscription.deleted"
+	chargeDisputeCreated        = "charge.dispute.created"
```

Comment:
Why do we the constants here and are not using them in the impl? Just asking if that is intentional (could imagine).

Reply:
Yeah makes sense to use them in the handler switch also. I'll raise a follow-up.